internal/command/docker.go

```diff
@@ -19,13 +22,19 @@ const (
 	fyneBin = "/usr/local/bin/fyne"
 	// gowindresBin is the path of the gowindres binary into the docker image
 	gowindresBin = "/usr/local/bin/gowindres"
+	// registry is the docker registry to use to pull images
+	registry = "docker.io"
 )
 
 // CheckRequirements checks if the docker binary is in PATH
 func CheckRequirements() error {
-	_, err := execabs.LookPath("docker")
+	_, err := exec.LookPath("docker")
```

Comment:
Could you please switch back to `execabs` (applies to more places in this PR)? It is being used because `exec` has a security issue on Windows that can't be fixed without breaking the API stability guarantee.